src/components/_video/VideoPlayer/VideoPlayer.tsx

```diff
@@ -416,13 +416,16 @@ const VideoPlayerComponent: React.ForwardRefRenderFunction<HTMLVideoElement, Vid
   }, [currentVolume, volumeToSave, player, setCachedVolume])
 
   // button/input handlers
-  const handlePlayPause = () => {
+  const handlePlayPause = useCallback(() => {
+    if (playerState === 'error') {
```

Comment:
```suggestion
    if (playerState === 'error' || playerState === 'ended') {
```
This is happens also when a video is ended. I think it makes sense to refactor this line

Reply:
Rafał just point me out that applying this suggestion will break the replay button on controls.

Anyway, I got confirmation from Adam that we shouldn't be able to replay a video by clicking on the background when a video is ended. We need to fix this at some point as well. 
I'll create a new issue just for this particular bug because it will require more changes to the code.